generator/internal/rust/templates/common/message.mustache

```diff
@@ -338,6 +338,27 @@ impl serde::ser::Serialize for {{Codec.Name}} {
         state.end()
     }
 }
+
+#[doc(hidden)]
```

Comment:
I think we want to leave these in the documentation, so people know that they can use `{message:?}`, e.g.

https://docs.rs/google-cloud-storage/0.25.0-preview3/google_cloud_storage/model/struct.Bucket.html#impl-Default-for-Bucket